src/test/unit/controls/PairedResult/PairedResultLoad-spec.js

```diff
@@ -1077,6 +1077,65 @@ describe("Paired Result - Load", () => {
             );
             expect(iconSVG.getAttribute("style")).toBe(`fill: ${COLORS.BLUE};`);
         });
+        it("loads only line if user sets showLine to be true and showShape to be false", () => {
```

Comment:
Could we add the describe block here for conditions?
```js
            describe("if user sets showLine to be true and showShape to be false", () => {
                it("loads only line", () => {
                    const input = getInput(valuesDefault, false, false);
                    input.legendOptions = {
                        showShape: false,
                        showLine: true
                    };
                    graphDefault.loadContent(new PairedResult(input));
                    const legendItem = fetchElementByClass(
                        pairedResultGraphContainer,
                        styles.legendItem
                    );
                    const svgElements = legendItem.querySelectorAll("svg");
                    const lineSVG = svgElements[0];
                    expect(svgElements.length).toBe(1);
                    expect(lineSVG).not.toBeNull();
                    expect(lineSVG.classList).toContain(styles.legendItemLine);
                });
            });
```

Reply:
Updated spec